### DISCUSSION TITLE
想参与 5.0 版本的开发吗？来报名吧！😎

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#2140] [Announcements]
大家好，近期我们发布了4.3  版本，你可以在这里找到发布信息：[NotionNext-Release](https://github.com/tangly1024/NotionNext/releases) 🎉

但 版本还有很多待完善的功能，我们非常期待能与社区的伙伴们一起共同构建 NotionNext 的生态，与社区一起发布最终的版本。为此，我们希望能够吸引更多对此感兴趣的同学们加入到 NotionNext 新版本的开发中来🥳。

如果你对参与这些任务感兴趣，可以在讨论区看到内容，[待修复的bug](https://github.com/tangly1024/NotionNext/discussions/categories/q-a) ， [待加入的功能](https://github.com/tangly1024/NotionNext/discussions/categories/ideas)

# 领取方式
领取任务的方式很简单，只需要在本帖下回复你想领取的任务链接，并提供你能够开始任务的最早时间，我们会按照报名的顺序及时间安排来决定参与的人选（contributor 优先），并将其添加到开发者群组中。

# 参与要求
请确保你有足够的开发时间。我们期望任务负责人能在一周内开始，在一个月内完成领取功能 80% 以上的开发工作。
请确保你能够进行基本的后续维护工作。也就是说，我们期望你能在接下来的半年内负责你所开发功能的 bug 修复和社区维护工作。

#你将获得
参与到社区最流行的微前端解决方案之一的全新版本建设中🤳
成为 NotionNext team member，将参与后续NotionNext商业计划的分成🤠

# 任务工作流

- 开发工作将在[next](https://github.com/tangly1024/NotionNext/tree/next)分支展开

- 任务领取后，task assign 给相关同学
- task 内沟通设计思路 & 方案
- 提交 pr
- 评审通过、合并 pr

希望能够吸引到更多的开发者加入到 NotionNext 开发中来，共同推动项目的发展！期待你的加入~

**Comment by qujingyang28** [comment]
很想参与无奈水平达不到
